### PR TITLE
Add faq to debug when chaos not working

### PR DESCRIPTION
**Pull request by zhouqiang-cl**
### What problem does this PR solve? 
Add faq to debug when chaos not working
### What is changed and how does it work?
Add faq to debug when chaos not working
### Check List 

Tests 

 - No code

Code changes

 - No code change

Side effects

 - None

Related changes

 - None

### CODE REVIEW

File: doc/faqs.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # FAQs
 
+## Debug
+
+### Q: Experiment not working after apply chaos
+You can following the steps to debug:
+- First using `kubectl describe` the chaos experiment resource. If see `NextStart`  and `NextRecover`  field in spec, then after `NextStart` the chaos will trigger.
+- Then use `kubectl logs -f -n chaos-testing chaos-controller-manager-xxxxx(replace this will controller-manager's pod name)`  to get controller-manager's log and see whether there are errors in it, If there are some related errors, please file an issue.
+- If not the above two cases, then you can use `kubectl get pods -n yourNamespace(using your namespace) --show-labels` to show the labels and check if it is meet selector.
```
Comment (cwen0) [inline doc/faqs.md]: If the pods didn't meet selector, `no pod is selected`  error message will be printed at the controller log

File: doc/faqs.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
 # FAQs
 
+## Debug
+
+### Q: Experiment not working after apply chaos
+You can following the steps to debug:
+- First using `kubectl describe` the chaos experiment resource. If see `NextStart`  and `NextRecover`  field in spec, then after `NextStart` the chaos will trigger.
+- Then use `kubectl logs -f -n chaos-testing chaos-controller-manager-xxxxx(replace this will controller-manager's pod name)`  to get controller-manager's log and see whether there are errors in it, If there are some related errors, please file an issue.
```
Comment (cwen0) [inline doc/faqs.md]: `kubectl logs -n chaos-testing chaos-controller-manager-xxxxx(replace this will controller-manager's pod name) | grep "ERROR"` ?

Comment (zhouqiang-cl) [reply doc/faqs.md]: greet

**Comment by zhouqiang-cl** [comment]
@cwen0 @YangKeao PTAL again

**Comment by cwen0** [comment]
@dcalvin PTAL

File: doc/faqs.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # FAQs
 
+## Debug
+
+### Q: Experiment not working after apply chaos
+You can following the steps to debug:
```
Comment (mahjonp) [inline doc/faqs.md]:
```suggestion

You can following the steps to debug:
```

File: doc/faqs.md [hunk 4]
```diff
@@ -1,5 +1,19 @@
...
+
+### Q: Experiment not working after chaos is applied
+
+You can debug as decribed below:
+
+Use `kubectl describe` to show the specified chaos experiment resource.
+
+- If there are `NextStart` and `NextRecover` fields under `spec`, then the chaos will be triggered after `NextStart` is executed.
+
+- If there are no `NextStart` and `NextRecover`fields under in `spec`, then use `kubectl logs -n chaos-testing chaos-controller-manager-xxxxx (replace this with the name of the controller-manager) | grep "ERROR"` to get controller-manager's log and see whether there are errors in it. For error message `no pod is selected`, use `kubectl get pods -n yourNamespace --show-labels` to show the labels and check if the selector is desired. For other related errors in controller's log, please file an issue.
```
Comment (dcalvin) [inline doc/faqs.md]:
```suggestion
- If there are no `NextStart` and `NextRecover`fields in `spec`, then use `kubectl logs -n chaos-testing chaos-controller-manager-xxxxx (replace this with the name of the controller-manager) | grep "ERROR"` to get controller-manager's log and see whether there are errors in it. For error message `no pod is selected`, use `kubectl get pods -n yourNamespace --show-labels` to show the labels and check if the selector is desired. For other related errors in controller's log, please file an issue.
```

**Pull request merged**